lib/Driver/ToolChains.cpp

```diff
@@ -124,6 +124,7 @@ static void addCommonFrontendArgs(const ToolChain &TC,
 
   inputArgs.AddAllArgs(arguments, options::OPT_I);
   inputArgs.AddAllArgs(arguments, options::OPT_F);
+  inputArgs.AddAllArgs(arguments, options::OPT_Fsystem);
```

Comment:
Okay. That doesn't match Clang's behavior, but it is a simple story and a defensible position.